src/Styleguide/Artwork/Banner.tsx

```diff
@@ -1,22 +1,62 @@
 import React from "react"
 import styled from "styled-components"
+import {
+  flexDirection,
+  alignItems,
+  alignContent,
+  justifyContent,
+  space,
+} from "styled-system"
 import { Avatar } from "../Elements/Avatar"
+import { Sans, Serif } from "../Elements/Typography"
 
-const Flex = styled.div`
+interface FlexProps {
+  flexDirection?: "row" | "column"
+  alignItems?: "flex-start" | "flex-end" | "center" | "baseline" | "stretch"
+  alignContent?:
+    | "flex-start"
+    | "flex-end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "stretch"
+  justifyContent?:
+    | "flex-start"
+    | "flex-end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "space-evenly"
+  m?: string
+  mr?: string
+  ml?: string
+  mt?: string
+  mb?: string
+}
+
+const Flex = styled.div.attrs<FlexProps>({})`
   display: flex;
-  flex-direction: row;
+  ${flexDirection};
+  ${alignItems};
+  ${alignContent};
+  ${justifyContent};
+  ${space};
 `
 
 export class Banner extends React.Component {
   render() {
     return (
-      <Flex>
+      <Flex flexDirection="row">
         <Avatar size="110px" src="https://picsum.photos/110/110/?random" />
-        <div>
-          <div>In show</div>
-          <div>Francesca DiMattio: Boucherouite</div>
-          <div>Salon 94</div>
-        </div>
+        <Flex flexDirection="column" justifyContent="center" ml="20px">
```

Comment:
Ahh -- `ml='20px'` -- can we stick to [palette's `space` values](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L89) at all times and religiously avoid one-off's? I spoke to brian and he said that in instances where the pixel values slightly diverge between what is in the zeplin redlines and what is in the design that we should round. (Also, at this point, any time an artibitrary `px` value is seen it should be considered a code smell, and in styled-component classes we should always be using the `getTheme()` styled-system helper and using constants.)